public/scss/application.scss

```diff
@@ -25,6 +25,24 @@
     > div:nth-child(3n+4) {
       clear: left;
     }
+    .remote-vetting-selector{
```

Comment:
Considering my previous review entry. Removing the 'remote-' portion from this scss decoration should be all it takes.